YACS-RCOS/yacs.n
# 294 - breadcrumb nav on explore pages

**Issue**

issue #294 
"When I am on a course page or subject page I want to know relatively where I am on the YACS page."

*Example:*
Added breadcrumb navigation on all subpages of /explore

**Test Procedure**

Go to YACS home -> explore, notice the row below the header that tells you where you are relative to the rest of the YACS pages

**Photos**

![Screen Shot 2020-11-15 at 9 02 01 PM](https://user-images.githubusercontent.com/44211696/99205233-e9b4b580-2785-11eb-965f-138295bc35d1.png)
